cmakegen/gen.go

```diff
@@ -39,7 +39,7 @@ func dependenciesToCmake(dependencies map[string]string) ([]Dependency, error) {
 	for n, v := range dependencies {
 		m, err := manifest.Load(filepath.Join(config.LibCachePath(n, v), "c3pm.yml"))
 		if err != nil {
-			return nil, err
+			continue
```

Comment:
Why would you change that?

Reply:
This function was throwing an error if a dependency in the c3pm.yml was not installed on the user computer.
By letting the return, the build and add command fail if you add a library by hand in the c3pm.yml.